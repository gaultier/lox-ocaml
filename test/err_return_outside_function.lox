return 1; // expect: Cannot return outside of a function body. Returning: `(Literal (Number 1) 1)`
