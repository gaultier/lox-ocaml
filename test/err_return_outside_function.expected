Cannot return outside of a function body. Returning: `(Literal (Number 1) 1)`
